src/unpage/agent/templates/blank.yaml

```diff
@@ -17,3 +17,28 @@ prompt: >
 #
 tools:
   - "*"
+
+# Customize and override configuration for any plugins the agent uses.
+# config:
+#   plugins:
+#     shell:
+#       enabled: true
+#       settings:
+#         tools:
+#           - handle: grep_syslog
+#             description: Search system log file
+#             command: grep -i {search_term} /var/log/syslog | tail -20
+#             args:
+#               search_term: Phrase to search for in the syslog file
```

Comment:
It also just occurred to me that we're only shallow-merging plugins, so if they specify e.g. shell tools here, they won't inherit the global ones. We should probably [deepmerge](https://pypi.org/project/deepmerge/) them instead (in another PR, probably).